auth/src/main/res/values-sw480dp/bools.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think adding a boolean resource to FirebaseUI for this is a step too far.  The developer should add this in their own app and then tell us to lock the orientation accordingly.

Reply:
I understood the feature wrong, sorry for the inconvenience.
I hope it's alright now, thanks for the guidance.